WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/viewholders/ErrorViewHolder.kt

```diff
@@ -3,13 +3,19 @@ package org.wordpress.android.ui.stats.refresh.lists.viewholders
 import android.view.ViewGroup
 import android.widget.TextView
 import org.wordpress.android.R
-import org.wordpress.android.R.layout
 import org.wordpress.android.ui.stats.refresh.lists.Error
 
-class ErrorViewHolder(parent: ViewGroup) : BaseStatsViewHolder<Error>(parent, layout.stats_error_view) {
+class ErrorViewHolder(val parent: ViewGroup) : BaseStatsViewHolder<Error>(parent, R.layout.stats_error_view) {
     private val title: TextView = itemView.findViewById(R.id.error_message)
     override fun bind(item: Error) {
         super.bind(item)
-        title.text = item.errorMessage
+        if (item.errorMessage.isNotBlank()) {
```

Comment:
Good change. Out of scope: I think this error view holder still needs some polishing (maybe we should add a refresh button to the block 🤷‍♂️ ).

Reply:
I agree. This is just a minimum that should be there so that we don't show empty cards (when a device is offline, for example). We should overhaul the error management in a separate task/PR that will also detect offline mode and displays a single error message.